docs/guides/Custom_Native_Nodes.md

```diff
@@ -0,0 +1,133 @@
+# Custom Native Nodes
+
+One of Elementary's primary goals is to provide a comprehensive vocabulary of
+native audio processing nodes from which functional, declarative composition
+can derive a huge set of higher level audio processes. But sometimes that's not
+quite enough; sometimes you will need particular audio processing primitives
```

Comment:
```suggestion
quite enough; sometimes you will need audio processing primitives
```

This would reads a bit clearer to me without "particular". It's clear from the context that the primitives are something you need that Elementary doesn't provide.